cirq-superstaq/cirq_superstaq/service.py

```diff
@@ -139,15 +147,17 @@ def __init__(
             EnvironmentError: if the `api_key` is None and has no corresponding environment
                 variable set.
```

Comment:
```suggestion
            EnvironmentError: If an API key was not provided and could not be found.
```
and same in qiskit-superstaq (i can't make suggestions on that line)